### ISSUE TITLE
Lock-ness is only polled for PICKUP_ALL

### ISSUE THREAD

**Issue by Draylar** [vini2003/Spinnery#32]
~~Lockness monster~~

**Version:** 1.15.2
**Issue Description: ** `isLocked`** is _only_ polled for `PICKUP_ALL` in `BaseContainer`, which means most actions work regardless of whether or not a slot is locked.
**Replication:**
1. create a simple container
2. add the following code in constructor
```java
for (WAbstractWidget widget : mainInterface.getWidgets()) {
    if (widget instanceof WSlot) {
        ((WSlot) widget).setLocked(true);
    }
}
```
3. attempt to interact with any slot in the inventory

**Comment by vini2003** [comment]
fixed in 1.15.2

1.16.3 soon

**Comment by vini2003** [comment]
done
